plugins/modules/flatpak.py

```diff
@@ -329,13 +329,34 @@ def _match_flat_using_flatpak_column_feature(module, binary, parsed_name, method
             return row.split()[0]
 
 
+def _is_flatpak_id(part):
```

Comment:
How about adding a comment here linking to the offical guidelines?

Reply:
It's in the new changes. I placed it inside the function with the links to the Flatpak and Flathub documentation about IDs.